bugbug/tools/code_review.py

```diff
@@ -56,25 +56,37 @@ class HunkNotInPatchError(ModelResultError):
     """Occurs when the hunk in the model result is not part of the patch."""
 
 
-PROMPT_TEMPLATE_SUMMARIZATION = """You are an expert reviewer for the Mozilla Firefox source code, with experience on source code reviews.
+TARGET_SOFTWARE = None
+
+PROMPT_TEMPLATE_SUMMARIZATION = (
+    """You are an expert reviewer for"""
+    + (f" the {TARGET_SOFTWARE}" if TARGET_SOFTWARE is not None else "")
+    + """ source code, with experience on source code reviews.
 
 Please, analyze the code provided and report a summarization about the new changes; for that, focus on the coded added represented by lines that start with "+".
 
 {patch}"""
+)
 
-PROMPT_TEMPLATE_REVIEW = """You will be given a task to generate a code review for the patch below. Use the following steps to solve it:
+PROMPT_TEMPLATE_REVIEW = (
+    """You will be given a task to generate a code review for the patch below. Use the following steps to solve it:
 1. Understand the changes done in the patch by reasoning about the summarization as previously reported.
 2. Identify possible code snippets that might result in possible bugs, major readability regressions, and similar concerns.
-3. Reason about each identified problem to make sure they are valid. Have in mind, your review must be consistent with the source code in Firefox.
+3. Reason about each identified problem to make sure they are valid. Have in mind, your review must be consistent with the source code"""
+    + (f" in {TARGET_SOFTWARE}" if TARGET_SOFTWARE is not None else "")
+    + """.
 4. Filter out comments that focuses on documentation, comments, error handling, tests, and confirmation whether objects, methods and files exist or not.
-5. Final answer: Write down the comments and report them using the JSON format provided below as valid comment examples (return only a JSON array).
+5. Filter out comments that are descriptive and filter out comments that are praising (example: "This is a good addition to the code.").
```

Comment:
Just realized now: should we split this into two (one about descriptive and one about praising), or keep only one?

Reply:
I can split it to make sure the model can focus on both

Reply:
done